doc/autoyast.md

```diff
@@ -2,92 +2,124 @@
 
 ## About this document
 
-This document describes which components are involved and how they cooperate in
-order to create a partitioning proposal for AutoYaST.
+This document describes which components are involved and how they cooperate to
+create a partitioning proposal for AutoYaST.
 
-## Use Cases
+## Use Cases Levels
 
-When it comes to partitioning, AutoYaST use cases can be categorized in three
+When it comes to partitioning, we can categorize AutoYaST use cases into three
 different levels:
 
 1. Automatic partitioning. The user does not care about the partitioning and
    trusts in AutoYaST to do the right thing.
-2. Semi-automatic partitioning. The user would like to set basic some
-   settings. E.g., a user would like to use LVM but has no idea how the devices
-   are partitions have to be set.
-3. Expert partitioning. The user specifies how the layout should look like.
-   Anyway, a complete definition is not required and some values could be missing
-   (AutoYaST should propose a reasonable default).
+2. Semi-automatic partitioning. The user would like to set some basic settings.
+   E.g., a user would like to use LVM but has no idea about how to configure
+   devices and partitions.
+3. Expert partitioning. The user specifies how the layout should look.
+   However, a complete definition is not required, and AutoYaST should propose
+   reasonable defaults for missing parts.
```

Comment:
I am not sure, but is it similar to also installation where is 1. next, 2. guided proposal and 3.expert partitioner?

Reply:
Yes, it is similar. The main difference is that in level 3, AutoYaST is able to figure out a lot of stuff by itself. For instance, this is enough:

```xml
<partitioning config:type="list">
  <drive>
    <device>/dev/sda</device>
    <use>all</use>
  </drive>
</partitioning>
```